modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1822,6 +1830,43 @@ private GroupByStep _getOneToManyObjectEntriesGroupByStep(
 		);
 	}
 
+	private Serializable _getOutputValue(String outputType, Object value) {
+		if (StringUtil.equals(
+				outputType, ObjectFieldConstants.BUSINESS_TYPE_BOOLEAN)) {
+
+			return GetterUtil.getBoolean(value);
+		}
+
+		if (StringUtil.equals(
+				outputType, ObjectFieldConstants.BUSINESS_TYPE_DATE)) {
+
+			DateFormat dateFormat = DateFormatFactoryUtil.getSimpleDateFormat(
```

Comment:
Hey @seltong,
I think we should format considering the user locale. I search a little and found this:
`DateFormat dateFormat = DateFormatFactoryUtil.getDate(locale);`
You can search for other things...
What do you think? Could you please test?